addons/website_event/static/src/scss/event_templates_list.scss

```diff
@@ -66,6 +66,33 @@
             &.left{
                 left: $card-spacer-x;
             }
+
+            &.o_wevent_date_with_flag {
+                $o_wevent_date_height: 0.8rem;
+                $o_wevent_flag_height: 20px;
+                .o_wevent_event_country_flag {
+                    height: $o_wevent_flag_height;
+                    margin-top: 2px;
+                    i {
+                        font-size: $o_wevent_flag_height;
+                        height: $o_wevent_flag_height;
+                    }
+                    img {
+                        height: $o_wevent_flag_height;
+                        max-width: 2.4rem;
+                        vertical-align: baseline;
+                    }
+                }
+                .o_wevent_event_date_text {
+                    line-height: $o_wevent_date_height;
+                }
+                .o_wevent_event_day, .o_wevent_event_month {
+                    @include font-size($o_wevent_date_height);
```

Comment:
I'm not sure if we're trying to carefully select a size so that it fits the flag size, but it doesn't seem to work as the used font does not have the same size for every character (there is a technical name for that but I don't remember what it is ^^):

![image](https://github.com/user-attachments/assets/ce8eeedd-8e08-43ef-80ae-d5b8ff75efa4)